packages/@react-aria/slider/stories/Slider.stories.tsx

```diff
@@ -1,121 +1,176 @@
 import {action} from '@storybook/addon-actions';
 import {ErrorBoundary} from '@react-spectrum/story-utils';
 import React from 'react';
-import {storiesOf} from '@storybook/react';
 import {StoryMultiSlider, StoryThumb} from './StoryMultiSlider';
 import {StoryRangeSlider} from './StoryRangeSlider';
 import {StorySlider} from './StorySlider';
 
 let message = 'Your browser may not support this set of format options.';
 
-storiesOf('Slider (hooks)', module)
-  .addDecorator(story => (
-    <ErrorBoundary message={message}>{story()}</ErrorBoundary>
-  ))
-  .add(
-    'single',
-    () => <StorySlider label="Size" onChange={action('onChange')} onChangeEnd={action('onChangeEnd')} showTip />
-  )
-  .add(
-    'single with big steps',
-    () => <StorySlider label="Size" onChange={action('onChange')} onChangeEnd={action('onChangeEnd')} step={10} showTip />
-  )
-  .add(
-    'single with origin',
-    () => <StorySlider label="Exposure" origin={0} minValue={-5} maxValue={5} step={0.1} onChange={action('onChange')} onChangeEnd={action('onChangeEnd')} showTip />
-  )
-  .add(
-    'single with aria label',
-    () => <StorySlider aria-label="Size" onChange={action('onChange')} onChangeEnd={action('onChangeEnd')} showTip />
-  )
-  .add(
-    'range',
-    () => (<StoryRangeSlider
-      label="Temperature"
-      defaultValue={[25, 75]}
-      onChange={action('onChange')}
-      onChangeEnd={action('onChangeEnd')}
-      showTip
-      formatOptions={{
+export default {
+  title: 'Slider (hooks)',
+  decorators: [(story) => <ErrorBoundary message={message}>{story()}</ErrorBoundary>]
+};
+
+export const Single = () => (
+  <StorySlider
+    label="Size"
+    onChange={action('onChange')}
+    onChangeEnd={action('onChangeEnd')}
+    showTip />
+);
+
+Single.story = {
+  name: 'single'
+};
```

Comment:
while you can do it this way, we've been doing it with an object declaration, lets keep with that
```
Single = {
  render...
  name...
}
```

Reply:
right, the current code style is from the codemod, figured I'd get everything converted first so I can get the storybook axe + interaction test stuff in and then change the style to match our other stories